ci-operator/step-registry/upi/conf/vsphere/zones/upi-conf-vsphere-zones-commands.sh

```diff
@@ -186,6 +186,28 @@ networking:
 EOF
 fi
 
+if [ ${MASTER_REPLICAS} -ne 0 ] && [ ${WORKER_REPLICAS} -ne 0 ]; then
```

Comment:
1. if use `if` block here, pls add the leading white spaces for the following lines.
2. Suggest to use two `if` blocks to judge `MASTER_REPLICAS` and `WORKER_REPLICAS` case separately.

Reply:
thanks for advice, only worker nums can be set to 0 when compact cluster scenario, so finally i add a if block for worker part.